src/dashboards/Compact/styles.scss

```diff
@@ -105,9 +105,26 @@
         transition: bottom 0.3s;
     }
 
+    .react-grid-item:matches(#scooter) > .react-resizable-handle::after {
+        content: '';
+        position: absolute;
+        width: 20%;
+        height: 0.25rem;
+        bottom: 0;
+        margin: 0 40%;
```

Comment:
Kan du bruke rem-verdi på margin? Litt vanskelig å forstå kva dette er 40% av.

Reply:
Fordelen med prosent på margin er at handle blir plassert riktig uavhengig av hvor mange kolonner det er. Hvis man setter det til rem blir det riktig for et sett med kolonner, men feil med en gang man legger til eller fjerner noen kort.

Reply:
Det forsvinner forsåvidt også når endringene for å justere kort i bredden kommer, håndtaket må justeres for det uansett 🤫